t/10-tests_overview.t

```diff
@@ -233,8 +233,8 @@ $t->get_ok('/tests/overview?distri=opensuse&version=13.1&groupid=1001&groupid=10
 $summary = get_summary;
 like(
     $summary,
-    qr/Summary of opensuse, opensuse test/i,
-    'multiple groups with no build specified yield latest build of first group'
+    qr/Summary of opensuse, opensuse test build 0091 /i,
+    'multiple groups with no build specified yield latest build of every group'
```

Comment:
It would probably make sense to add a test with groups that result in two different build names. Right now the result is like before, so the test doesn't prove that the change works.

Reply:
Done